torch/_functorch/aot_autograd.py

```diff
@@ -2780,6 +2780,10 @@ class CompiledFunction(torch.autograd.Function):
         metadata = fw_metadata
         num_symints_saved_for_bw = _num_symints_saved_for_bw
 
+        @staticmethod
+        def _compiled_autograd_key(ctx):
+            return (aot_config.aot_id, *ctx.symints)
```

Comment:
blegh, fine. When I originally added `aot_id` it was purely a debugging aid but I guess we can make it load bearing.

Reply:
Haha, open to suggestions here.